### PR TITLE
Db 1199 part 1

### PR DESCRIPTION
**Pull request by alburde1**
- Changed a lot of variable names from camelCase to underscored (only in functions I was already updating for the actual changes)
- Removed many functions from errorInterface.py (some completely factored out, some moved to function_bag.py) and updated related function calls to use the function_bag functions instead of relying on the interface.
- Updated several calls to directly look up from the dicts instead of calling functions to do so
- Replaced several functions to just query using sqlalchemy directly, eliminating extra function calls and levels of obscurity
- Connected all updated functions to GlobalDB.db().session instead of specific error interfaces

### CODE REVIEW

File: dataactcore/interfaces/function_bag.py [hunk 1]
```diff
@@ -243,3 +246,78 @@ def addJobsForFileType(fileType, filePath, filename, submissionId, existingSubmi
...
+    fileRec.file_status_id = FILE_STATUS_DICT[ValidationError.getErrorTypeString(error_type)]
+    if extra_info is not None:
+        if "missing_headers" in extra_info:
+            fileRec.headers_missing = extra_info["missing_headers"]
+        if "duplicated_headers" in extra_info:
+            fileRec.headers_duplicated = extra_info["duplicated_headers"]
+
+    sess.add(fileRec)
+    sess.commit()
+    return True
```
Comment (cmc333333) [inline dataactcore/interfaces/function_bag.py]: I know this return value existed before you tackled this code, but if we're reworking this function now -- does it make sense to remove the `return`? It doesn't seem to be providing any value (the only other way out of this function is an exception).

File: dataactcore/interfaces/function_bag.py [hunk 2]
```diff
@@ -243,3 +246,93 @@ def addJobsForFileType(fileType, filePath, filename, submissionId, existingSubmi
...
+        filename: name of error report in S3
+
+    Returns:
+        True if successful
+    """
+    sess = GlobalDB.db().session
+    fileComplete = createFileIfNeeded(job_id, filename)
+    fileComplete.file_status_id = FILE_STATUS_DICT['complete']
+    sess.commit()
+    return True
```
Comment (cmc333333) [inline dataactcore/interfaces/function_bag.py]: Similar note here about return value. In both cases, we'd want to update the docstring, too.

File: dataactcore/interfaces/function_bag.py (hunk 1 above)
Comment (alburde1) [inline dataactcore/interfaces/function_bag.py]: Updated this and the below comment

File: dataactbroker/handlers/fileHandler.py [hunk 3]
```diff
@@ -339,7 +339,7 @@ def getStatus(self):
             submission = self.jobManager.getSubmissionById(submission_id)
 
             # Check that user has access to submission
-            user = self.checkSubmissionPermission(submission)
```
Comment (cmc333333) [inline dataactbroker/handlers/fileHandler.py]: Good call

**Comment by cmc333333** [comment]
Excellent work

**Pull request merged**